src/main/twirl/uk/gov/hmrc/play/views/helpers/singleCheckbox.scala.html

```diff
@@ -19,13 +19,13 @@
 @import views.html.helper._
 
 @elements = @{ new FieldElements(field.id, field, null, args.toMap, messages) }
-@value = @{ field.value match { case Some(x) => x case None => "false" case x => x }}
-<div class="form-field-single @if(elements.hasErrors) {error}">
+@value = @{ field.value match { case Some(x) => x case None => "false" case x => x } }
+@style = @{ elements.args.get('_inLine) match { case Some(true) => "form-checkbox" case _ => "block-label" } }
```

Comment:
I'm not sure we want to offer the ability to choose. They should always be a `block-label`. @feedmypixel @gavinwye any thoughts on this?